planner_reasoner/lib/planner_reasoner.pl

```diff
@@ -87,6 +87,13 @@
                                         SubSubIndent is SubIndent+10,
                                         format(Out, '~*|(~w', [SubSubIndent,PredName1]),
                                         generate_precondition0(SubPredicateList, SubIndent, Out));
+  (rdf(H1, rdf:type, pp:'And') ->
```

Comment:
Head and Tale of a list, but I will anyway re-implement this part once there is some kind of stable PDDL ontology